arangod/Cluster/Maintenance.cpp

```diff
@@ -496,6 +496,7 @@ VPackBuilder getShardMap(VPackSlice const& collections) {
     // But then shardMap can also be empty, so we are good.
     if (collections.isObject()) {
       for (auto collection : VPackObjectIterator(collections)) {
+        TRI_ASSERT(collection.value.isObject());
```

Comment:
```suggestion
        TRI_ASSERT(collection.value.isObject());
        if (collection.value.hasKey(SHARDS) && collection.value.get(SHARDS).isObject()) {
```

Reply:
Needs closing braket of course.

Reply:
Applied the suggestion, slightly differently.